### ISSUE TITLE
Test on a new speaker 

### ISSUE THREAD

**Issue by IsmaelAh** [Anwarvic/Speaker-Recognition#9]
when the test be on a speaker(audio file for speaker not from the 50 sp) it will give the nearest speaker ,how i can calculate the thresold that when a new speaker not in data ,the output must unknown.

**Comment by Anwarvic** [comment]
That's a good question, @IsmaelAh !!

The short answer is 0. zero is a good starter threshold value.

The long answer is a little bit complicated. First, we have to split the data into three sets: training set, validation set, and test set. Then we try different values for the threshold and choose the one that maximizes the validation set.
